_sass/components/_dropdown.scss

```diff
@@ -1,188 +1,11 @@
-$dropdown-min-width: 125px;
-
-.dropdown-arrow {
-  display: none;
-}
-
 @include at-media("mobile-lg") {
   .dropdown {
-    position: relative;
-    z-index: 1;
-
     li {
       border-bottom: $border-width solid $border-color;
 
       &:last-of-type {
         border: none;
       }
     }
-
-    ul {
```

Comment:
For context, the language picker dropdown is implemented as `<ul class="dropdown">`, so `.dropdown ul` doesn't match anything.